### PR TITLE
Add CCRCF435AIO

### PR DESCRIPTION
**Pull request by ASDosjani**
New AT32 aio with AM32.
image
image

### CODE REVIEW

**Comment by haslinghuis** [comment]
Should add these nice pictures here: http://betaflight.com/docs/category/boards/CCRCF435AIO

**Comment by haslinghuis** [comment]
Would be interesting to test with

    #define MAX_SUPPORTED_MOTORS 4

File: configs/CCRCF435AIO/config.h [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU        AT32F435G
+
+#define BOARD_NAME           CCRCF435AIO
+#define MANUFACTURER_ID      CCRC
```
Comment (haslinghuis) [inline configs/CCRCF435AIO/config.h:27]: We need a manufacturer ID

File: configs/CCRCF435AIO/config.h [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+#define USE_BARO
+#define USE_BARO_DPS310
+
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+
+#define USE_MAX7456
+
+#define USE_BEEPER
```
Comment (haslinghuis) [inline configs/CCRCF435AIO/config.h:43]: USE_BEEPER is already defined.

File: configs/CCRCF435AIO/config.h [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+#define USE_QUAD_MIXER_ONLY
+
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_AUTO
+#define DEFAULT_DSHOT_TELEMETRY DSHOT_TELEMETRY_ON
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define BEEPER_INVERTED
+#define SYSTEM_HSE_MHZ          8
+#define USE_SPI_GYRO
```
Comment (haslinghuis) [inline configs/CCRCF435AIO/config.h:99]: USE_SPI_GYRO is already defined

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/CCRCF435AIO/config.h (hunk 1 above)
Comment (ASDosjani) [reply configs/CCRCF435AIO/config.h:27]: This is, I asked the manufacturer he said CCRC is ok.

Comment (haslinghuis) [reply configs/CCRCF435AIO/config.h:27]: We need company name and website

Comment (haslinghuis) [reply configs/CCRCF435AIO/config.h:27]: https://github.com/betaflight/config/pull/250

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
> Would be interesting to test with
> 
> ```
> #define MAX_SUPPORTED_MOTORS 4
> ```

is this idea resolved?  i dont know what `USE_QUAD_MIXER_ONLY` does TBH, can `+` & `custom` layout still work?

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**